test/asmcup/runtime/RobotTest.java

```diff
@@ -0,0 +1,52 @@
+package asmcup.runtime;
+
+import asmcup.vm.VM;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+public class RobotTest {
+	Robot robot = null;
+
+	@Before
+	public void setUp() {
+		robot = new Robot(42);
+		robot.position(42, 21);
+	}
+
+	@Test
+	public void testAccelerometer() {
+		World world = new World();
+		// Read Accelerometer
+		VM vm = robot.getVM();
+		vm.push8(Robot.IO_ACCELEROMETER);
+		vm.setIO(true);
+		robot.handleIO(world);
+
+		// We did not drive, so we should expect 0 values.
+		float x = vm.popFloat();
+		float y = vm.popFloat();
+		assertTrue("X is 0", Math.abs(0.0f - x) < 0.001f);
+		assertTrue("Y is 0", Math.abs(0.0f - y) < 0.001f);
+		assertTrue("X == Y", Math.abs(x - y) < 0.001f);
```

Comment:
There's an [assertEquals](http://junit.sourceforge.net/junit3.8.1/javadoc/junit/framework/Assert.html#assertEquals(java.lang.String,%20float,%20float,%20float)) for floats that does exactly what you're trying to do here. :)